src/NServiceBus.AcceptanceTests/Sagas/When_using_ReplyToOriginator.cs

```diff
@@ -10,57 +10,83 @@
     public class When_using_ReplyToOriginator : NServiceBusAcceptanceTest
     {
         [Test]
-        public async Task Should_set_Reply_as_messageintent()
+        public async Task Should_preserve_correlation_context()
         {
+            var messageId = Guid.NewGuid().ToString();
```

Comment:
we don't need this variable. We can assign a guid to the message and then store the received correlation id into the context on the handler